src/app/shared/palette/order-summary/write-order-summary-field.component.ts

```diff
@@ -0,0 +1,40 @@
+import { Component, OnInit } from '@angular/core';
+import { AbstractFieldWriteComponent } from '../base-field/abstract-field-write.component';
+import { FormControl, FormGroup, FormArray } from '@angular/forms';
+
+@Component({
+  selector: 'ccd-write-order-summary-field',
+  templateUrl: './write-order-summary-field.html'
+})
+export class WriteOrderSummaryFieldComponent extends AbstractFieldWriteComponent implements OnInit {
+
+  paymentReference: FormControl;
+  paymentTotal: FormControl;
+  feesArray: FormArray;
+  orderSummaryGroup: FormGroup;
```

Comment:
I don't believe exposing those variables at class level is needed, they could be kept in function-scope.